dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -17,6 +17,9 @@
 #include "PushNotificationChannel.h"
 #include "externs.h"
 #include <string_view>
+#include <frameworkudk/pushnotifications.h>
+#include "..\..\BuildOutput\Debug\x64\PushNotificationsLongRunningTask.ProxyStub\NotificationsLongRunningProcess_h.h"
```

Comment:
This should be `#include <NotificationsLongRunningProcess_h.h>`.

If there are compile issues, we need to talk offline. But I think I already did everything needed to avoid referencing this long path.

Reply:
Yeah this long path that inferences debug and x64 specifically cannot be right. Please fix.

Reply:
I noticed that I didn't add the references to the Reunion DLL vcxproj to look at the Output directory of the LRP proxy stub. I just gave guidance to @sharath2727 to address this, it should be straightforward.

Reply:
done